modules/bitgo/src/v2/coins/near.ts

```diff
@@ -79,14 +130,16 @@ export class Near extends BaseCoin {
   protected readonly _staticsCoin: Readonly<StaticsBaseCoin>;
   constructor(bitgo: BitGoBase, staticsCoin?: Readonly<StaticsBaseCoin>) {
     super(bitgo);
-
     if (!staticsCoin) {
       throw new Error('missing required constructor parameter staticsCoin');
     }
 
     this._staticsCoin = staticsCoin;
   }
 
+  static initialized = false;
+  static MPC: Eddsa;
```

Comment:
nit: these should probably be `protected`

```suggestion
  static initialized = false;
  static MPC: Eddsa;
```

Reply:
Done